mailpoet/lib/Tracy/DIPanel/DIPanel.php

```diff
@@ -60,10 +61,11 @@ private function getServicesFromContainer($name) {
     $containerWrapper = ContainerWrapper::getInstance();
     $reflection = new \ReflectionProperty(ContainerWrapper::class, $name);
     $reflection->setAccessible(true);
+    /** @var Container $container */
     $container = $reflection->getValue($containerWrapper);
     $reflection = new \ReflectionProperty(get_class($container), 'services');
     $reflection->setAccessible(true);
-    return [$reflection->getValue($container), $container->getDefinitions()];
+    return [$reflection->getValue($container), $container->getServiceIds()];
```

Comment:
@lysyjan It all works great but with this change, the Tracy panel does not populate the `Used In` column.

Is this required for the upgrade? Or is this to remove the PHPStan warnings?

I don't see it as blocking, we could create a separate ticket to investigate if you prefer.